process/datadog_checks/process/process.py

```diff
@@ -467,7 +475,7 @@ def _process_service_check(self, name, nb_procs, bounds, tags):
             "process.up",
             status,
             tags=service_check_tags,
-            message=message_str % (status_str[status], nb_procs, name)
+            message=message_str.format(status_str[status], nb_procs, name)
```

Comment:
Since `message_str` is used only here, we can just move the string down here:
```
message="PROCS {}: {} processes found for {}".format(status_str[status], nb_procs, name)
```